src/libAtomVM/term.h

```diff
@@ -537,6 +545,26 @@ static inline int term_is_function(term t)
     return 0;
 }
 
+/**
+ * @brief Checks if a term is a function reference
+ *
+ * @details Returns true if a term is a function reference such as "fun m:f/a", otherwise false.
+ * @param t the term that will be checked.
+ * @return true if check succeeds, false otherwise.
+ */
+static inline bool term_is_function_reference(term t)
```

Comment:
Not sure about this function name, I saw this called both function reference, and exported function in the official documentation. I would call this function reference that sounds clearer.

Reply:
`fun f/a` funs are actually implemented as anonymous functions, so yes, `is_function_external` is the right name.

Reply:
`fun f/a` funs are actually implemented as anonymous functions, so yes, `is_function_external` is the right name.